code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -295,10 +306,10 @@
 	if(!HAS_TRAIT(our_turf, TRAIT_RUSTY))
 		return
 
-	source.adjustBruteLoss(-4, FALSE)
-	source.adjustFireLoss(-4, FALSE)
-	source.adjustToxLoss(-4, FALSE, forced = TRUE)
-	source.adjustOxyLoss(-4, FALSE)
+	source.adjustBruteLoss(-5, FALSE)
+	source.adjustFireLoss(-5, FALSE)
+	source.adjustToxLoss(-5, FALSE, forced = TRUE)
+	source.adjustOxyLoss(-5, FALSE)
```

Comment:
Your experience here implies this would be best as a define or variable, so that you don't need to modify the same value in four places.